Containerfile

```
@@ -1,3 +1,5 @@
-FROM scratch
+FROM alpine/git
```

```suggestion
FROM docker.io/alpine/git
```

Just so things like podman don't have to guess based on their registries.conf.